### PR TITLE
feat: GEOT-156: invite friend

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

**Comment by ra8ga** [comment]
retest

**Comment by ra8ga** [comment]
retest please

File: packages/ui/InviteFriends/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import styles from './styles.scss'
+import cx from 'classnames'
+import { useTranslation } from '@gtms/commons/i18n'
+import { Button } from '@gtms/ui/Button'
+import useClipboard from 'react-use-clipboard'
+
+export const InviteFriends: FC<{
+  additionalStyles?: string
+}> = ({ additionalStyles }) => {
+  const [isCopied, setCopied] = useClipboard(
```
Comment (ra8ga) [inline packages/ui/InviteFriends/index.tsx:11]: should I write test for that?

File: packages/ui/InviteFriends/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    'http://jedziemyna.pl/openerfestival'
+  )
+  const { t } = useTranslation('inviteFriends')
```
Comment (ra8ga) [inline packages/ui/InviteFriends/index.tsx:14]: do we use useTranslations this way? should I write test for that?

**Comment by ra8ga** [comment]
please review it

File: packages/app-andrew/src/pages/_app.tsx [hunk 3]
```diff
@@ -9,10 +9,12 @@ class GTMSApp extends App {
     const { Component, pageProps } = this.props
 
     return (
-      <div>
+      <>
+        {' '}
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/_app.tsx:13]: this space is also needed? And I don't see a point to keep the comment below

File: packages/ui/InviteFriends/index.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+  return (
+    <>
+      <div
+        className={cx(styles.wrapper, additionalStyles)}
+        data-testid="invite-friends"
+      >
+        <h2>Zaproś znajomych, bla bla bla, bla bla bla</h2>
```
Comment (mariusz-kabala) [inline packages/ui/InviteFriends/index.tsx:22]: use translator everywhere

File: packages/ui/InviteFriends/index.tsx (hunk 1 above)
Comment (mariusz-kabala) [reply packages/ui/InviteFriends/index.tsx:11]: no

File: packages/ui/InviteFriends/index.tsx (hunk 2 above)
Comment (mariusz-kabala) [reply packages/ui/InviteFriends/index.tsx:14]: no

File: packages/app-andrew/src/pages/_app.tsx (hunk 3 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/_app.tsx:13]: once we broke log / register pages bacause of that I would left the comment here as it's the place where we can break a lot of pages, but not all of them so it's possible that someone will break something and won't notice

Comment (mariusz-kabala) [reply packages/app-andrew/src/pages/_app.tsx:13]: wtf, so css is broken if it depends on a space, and needs to be fixed. fix it, not leave a comment

File: README.md [hunk 5]
```diff
@@ -274,3 +274,11 @@ this is how you can use static files in css
     width: 100%;
 }
 ```
+
+# Adding external npm modules
```
Comment (mariusz-kabala) [inline README.md:278]: remove this comment, its not needed, you will start also explaining how lifecycle works in react? or how to write css code?

File: packages/app-andrew/src/pages/_app.tsx (hunk 3 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/_app.tsx:13]: here is that rule in css:

html,
body,
body > div:first-of-type {
  height: 100%;
}

this first div is <div id="__next">, if someone will add any wrapper component that creates <div> then height 100% won't be applied for container that keeps login page or register page height 100%, other solution is vh.

Comment (mariusz-kabala) [reply packages/app-andrew/src/pages/_app.tsx:13]: how this is related to the space? I was asking about `{' '}`

Comment (ra8ga) [reply packages/app-andrew/src/pages/_app.tsx:13]: no, this is to be removed, should I remove {' '} and left comment in line 14 ?

Comment (mariusz-kabala) [reply packages/app-andrew/src/pages/_app.tsx:13]: just remove it, no need for extra comments

Comment (ra8ga) [reply packages/app-andrew/src/pages/_app.tsx]: fixed

File: packages/ui/InviteFriends/index.tsx (hunk 4 above)
Comment (ra8ga) [reply packages/ui/InviteFriends/index.tsx]: fixed

File: README.md (hunk 5 above)
Comment (ra8ga) [reply README.md]: fixed

File: packages/ui/InviteFriends/index.tsx [hunk 6]
```diff
@@ -19,14 +19,14 @@ export const InviteFriends: FC<{
...
-        <h2>Zaproś znajomych, bla bla bla, bla bla bla</h2>
+        <h2>{t('invite.inviteFriends')}</h2>
         <Button
           additionalStyles={styles.btn}
           onClick={setCopied}
           data-testid="invite-friends-copy-button"
         >
-          {!isCopied && t('copyLink')}
-          {isCopied && 'skopiowano'}
+          {!isCopied && t('skopiujLink')}
```
Comment (mariusz-kabala) [inline packages/ui/InviteFriends/index.tsx:28]: translations keys only in english

Comment (ra8ga) [reply packages/ui/InviteFriends/index.tsx]: fixed

File: packages/ui/Navigation/index.tsx [hunk 7]
```diff
@@ -55,10 +56,12 @@ export const Navigation: FC<{}> = () => {
         {links.map((value, index) => {
           return (
             <li className={styles.link} key={index}>
-              <a>
-                <i>{value.icon}</i>
-                <span>{value.label}</span>
-              </a>
+              <Link href={value.url}>
```
Comment (mariusz-kabala) [inline packages/ui/Navigation/index.tsx:59]: this is react router component, it does not have `href` prop but `to`

File: packages/ui/NavigationDot/index.tsx [hunk 8]
```diff
@@ -42,7 +42,7 @@ export const NavigationDot: FC = () => (
   <ul className={styles.navigationDot} data-testid="navigationDot">
     {mockData.map((value) => (
       <li key={value.id}>
-        <Link href="/remind-password">
+        <Link href="/account">
```
Comment (mariusz-kabala) [inline packages/ui/NavigationDot/index.tsx:45]: the same here, no `href` prop, did you test it? does it work?

File: packages/ui/Navigation/index.tsx (hunk 7 above)
Comment (ra8ga) [reply packages/ui/Navigation/index.tsx:59]: We use import { Link } from '@gtms/commons/i18n'
so here: https://github.com/isaachinman/next-i18next I see <Link href not <Link to

File: packages/ui/NavigationDot/index.tsx (hunk 8 above)
Comment (ra8ga) [reply packages/ui/NavigationDot/index.tsx:45]: We use import { Link } from '@gtms/commons/i18n'
so here: https://github.com/isaachinman/next-i18next I see <Link href not <Link to

File: packages/ui/Navigation/index.tsx (hunk 7 above)
Comment (ra8ga) [reply packages/ui/Navigation/index.tsx:59]: are you sure there is to instead of href?

File: packages/ui/NavigationDot/index.tsx (hunk 8 above)
Comment (ra8ga) [reply packages/ui/NavigationDot/index.tsx:45]: are you sure there is to instead of href?

**Pull request merged**